### PR TITLE
feature/normalize_radius_username

### PR DESCRIPTION
**Pull request by jrouzierinverse**
Description
===========
Normalize the radius username for machine auth when using TLS

Impacts
=======
RADIUS authorize workflow

Delete branch after merge
=========================
NO

NEWS file entries
=================

Enhancements
------------
* Have the ability to normalize the machine auth username when using TLS

### CODE REVIEW

**Comment by louismunro** [comment]
The code is fine, but the documentation does not explain what the option is doing, or why one would want to use it. That should be made more explicit otherwise people will just toggle it on and off for no reason.

**Comment by jrouzierinverse** [comment]
Where in the documentation do you want to place it?

**Comment by louismunro** [comment]
Just in the help string.
The word "normalize" can mean anything depending on the context.
Just make it explicit what we mean by that.

**Comment by jrouzierinverse** [comment]
Updated documentation.
Provide any recommendation.

File: conf/documentation.conf [hunk 1]
```diff
@@ -1123,6 +1123,13 @@ description=<<EOT
 Ability to manage all active devices from a same switch port
 EOT
 
+[advanced.normalize_radius_machine_auth_username]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Use the RADIUS username instead of the TLS certificate common name when doing machine authentication.
```
Comment (louismunro) [inline conf/documentation.conf:1130]: Much better.

**Pull request merged**